dashboards doc edits (#17409)
netdata/netdata: docs/cloud/visualize/dashboards.md

  
  After you are done configuring the chart, you can also change the type of the chart from the right hand side of the [Title bar](https://github.com/netdata/netdata/blob/master/docs/cloud/visualize/interact-new-charts.md#title-bar), and select which of the final dimensions you want to be visible and in what order, from the [Dimensions bar](https://github.com/netdata/netdata/blob/master/docs/cloud/visualize/interact-new-charts.md#dimensions-bar).

- The **Add Text** button on the top right of the interface creates a new card with user-defined text, which you can use to describe or document a particular dashboard's meaning and purpose.

> ### Important
>
> Be sure to click the **Save** button any time you make changes to your dashboard.

## Using your dashboard

> Dashboards are designed to be interactive and flexible so you can design them to your exact needs. They are made from any number of charts and cards, which can contain charts or text.

Reviewer: ```suggestion
Dashboards are designed to be interactive and flexible so you can design them to your needs. They are made from any number of charts and cards, which can contain charts or text.
```

### Charts

The charts you add to any dashboard are [fully interactive](https://github.com/netdata/netdata/blob/master/docs/cloud/visualize/interact-new-charts.md), just like any other Netdata chart. You can zoom in and out, highlight timeframes, and more.

Charts also synchronize as you interact with them, even across contexts _or_ nodes.

### Text cards

You can use text cards as notes to explain to other members of the [War Room](https://github.com/netdata/netdata/blob/master/docs/cloud/manage/organize-your-infrastrucutre-invite-your-team.md#netdata-cloud-war-rooms) the purpose of the dashboard's arrangement.

> By clicking the `T` icon on the text box, you can cycle through font sizes.

Reviewer: ```suggestion
By clicking the `T` icon on the text box, you can switch between font sizes.
```

### Move elements

To move a chart or a card, click and hold on **Drag & drop** at the top right of each element and drag it to a new location. A green placeholder indicates the
new location. Once you release your mouse, other elements re-sort to the grid system automatically.

### Resize elements

To resize any element on a dashboard, click on the bottom-right corner and drag it to its new size. Other elements re-sort to the grid system automatically.

### Go to chart

Quickly jump to the location of the chart in either the [Metrics tab](https://github.com/netdata/netdata/blob/master/docs/dashboard/metrics-tab-and-single-node-tabs.md) or if the chart refers to a single node, its single node dashboard by clicking the 3-dot icon in the corner of any chart to open a menu. Hit the **Go to Chart** item.

You'll land directly on that chart of interest, but you can now scroll up and down to correlate your findings with other
charts. Of course, you can continue to zoom, highlight, and pan through time just as you're used to with Netdata Charts.

### Rename a chart

> By using the 3-dot icon in the corner of any chart, you can rename it to better explain your use case, or the visualization settings you have selected for the chart.

Reviewer: ```suggestion
Using the 3-dot icon in the corner of any chart, you can rename it to better explain your use case or the visualization settings you've chosen for the chart.
```
